### DISCUSSION TITLE
can't upload file

### DISCUSSION THREAD

**Discussion by christinaanne58** [apprenticeharper/DeDRM_tools#1484] [Q&A]
I'm using Calibre 5.9.0 and it won't upload file DeDRM_tools-7.0.0b3.  What am I doing wrong?

**Comment by ElleKayEm** [comment]
The file you need is DeDRM_tools_7.0.3.zip. Underscore after tools, not a hyphen. It is the first link under assets on [this page](https://github.com/apprenticeharper/DeDRM_tools/releases/tag/7.0.3).

**Comment by christinaanne58** [comment]
Hi
I appreciate the personal email, thanks!  I do have the right file (despite my typo) but I can’t make Calibre upload it.  Here are a few  screenshots — one showing that Calibre won’t upload when I select the file ….. and the other gives an error message when I select the zip file Calibre WILL upload.  I am a dunce!  I’ve used your Apprentice tools for years (and am a HUGE fan)  ….. what am I missing this time?
Thanks,
Chris Lotito

**Comment by ElleKayEm** [comment]
I didn't send an email -- just answered at GitHub.  I'm not seeing your screen shots so I'm not sure what file or what error message you're referring to.  The file with a hyphen instead of the underscore is what you get when you select the second link (source code) under Assets.  You need the first link/zip file with the underscore, and that should work.

**Reply by christinaanne58** [reply]
Here's my screenshots, if I uploaded them correctly!
[#1.pdf](https://github.com/apprenticeharper/DeDRM_tools/files/5825273/1.pdf)
[#2.pdf](https://github.com/apprenticeharper/DeDRM_tools/files/5825275/2.pdf)
[#3.pdf](https://github.com/apprenticeharper/DeDRM_tools/files/5825276/3.pdf)

**Comment by ElleKayEm** [comment]
You are still trying to load the wrong file(s).  Here is a direct link to the file you need:  [https://github.com/apprenticeharper/DeDRM_tools/releases/download/7.0.3/DeDRM_plugin_7.0.3.zip](https://github.com/apprenticeharper/DeDRM_tools/releases/download/7.0.3/DeDRM_plugin_7.0.3.zip)

Make sure that your browser isn't unzipping this on download.  Do not unzip it.  Load it as is into calibre.

**Reply by christinaanne58** [reply]
THANK YOU!  I had no idea my browser was unzipping the file as it downloaded.  
I had my husband download it onto his computer (Microsoft Edge) and then hand it off to me, and it installed PERFECTLY.
(I'm on a Mac using Safari as a browser, which I am now reconsidering!)
THANK YOU THANK YOU THANK YOU

**Reply by ElleKayEm** [reply]
I think there's a way to prevent Safari from doing that.  Either in preferences or holding some button while clicking.

**Reply by christinaanne58** [reply]
Yep, did that, thanks!  For anyone else having this problem, go to   Safari>Preferences>General   and uncheck the box that says "Open 'safe' files after downloading"
THANK YOU, ELLKAYEM!!!
